packages/react-native-bpk-component-animate-height/package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "react-native-bpk-component-animate-height",
-	"version": "2.0.57",
+	"version": "2.0.58",
```

Comment:
How come this has changed? They don't seem to be out of date on Master?

Reply:
My bad - they are outdated on master :)

Reply:
they are https://github.com/Skyscanner/backpack-react-native/blob/master/packages/react-native-bpk-component-animate-height/package-lock.json.

It's prob the same lerna problem we have in the main repo

Reply:
Cool. I've fixed them on master just to make the commit history more interpretable!